Sources/EpoxyCollectionView/CollectionView/Internal/ReuseIDStore.swift

```diff
@@ -25,18 +25,46 @@ public final class ReuseIDStore {
 
   /// Vends a new reuse identifier string for the given `ViewDifferentiator`, generating a new reuse
   /// identifier whenever a new `viewDifferentiator` is encountered as determined by its equality.
-  public func reuseID(for viewDifferentiator: ViewDifferentiator) -> String {
+  public func registerReuseID(for viewDifferentiator: ViewDifferentiator) -> String {
     if let existingReuseID = reuseIDsForViewDifferentiators[viewDifferentiator] {
       return existingReuseID
-    } else {
-      let viewType = viewDifferentiator.viewTypeDescription
-      let uniqueViewDifferentiatorCount = uniqueViewDifferentiatorCountsForViewTypes[viewType] ?? 0
-      uniqueViewDifferentiatorCountsForViewTypes[viewType] = uniqueViewDifferentiatorCount + 1
-
-      let reuseID = "\(viewType)_\(uniqueViewDifferentiatorCount)"
-      reuseIDsForViewDifferentiators[viewDifferentiator] = reuseID
-      return reuseID
     }
+
+    let viewType = viewDifferentiator.viewTypeDescription
+    let uniqueViewDifferentiatorCount = uniqueViewDifferentiatorCountsForViewTypes[viewType] ?? 0
+    uniqueViewDifferentiatorCountsForViewTypes[viewType] = uniqueViewDifferentiatorCount + 1
+
+    let reuseID = "\(viewType)_\(uniqueViewDifferentiatorCount)"
+    reuseIDsForViewDifferentiators[viewDifferentiator] = reuseID
+    return reuseID
+  }
+
+  /// Attempts to dequeue a reuse identifier string for the given `ViewDifferentiator`.
+  public func dequeueReuseID(for viewDifferentiator: ViewDifferentiator) -> String? {
```

Comment:
naming nit: we're not really dequeuing anything here, so not sure if that's the right name. Maybe just `reuseID` would suffice?

Reply:
Or: `public func registeredReuseID(for viewDifferentiator: ViewDifferentiator) -> String?`

Which might fit better with this naming suggestion https://github.com/airbnb/epoxy-ios/pull/28/files#r614463470